modules/webpack.config.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * webpack.config.js - webpack configuration script for testing ilib-localeinfo
+ *
+ * Copyright © 2022, JEDLSoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var webpack = require('webpack');
+var path = require('path');
+
+module.exports = {
+    mode: "development",
+    entry: "./node_modules/ilib-localeinfo",
```

Comment:
I think It's a bit uncommon way to give an entry to a directory under node_module. is it a general usage now?

Reply:
It is uncommon, yes. The problem is that there is no other code here to put at the top level of the webpack. Maybe I should make one?